src/main/java/com/offz/spigot/staminaclimb/Climbing/ClimbBehaviour.java

```diff
@@ -33,25 +33,31 @@
             "CHEST",
             "MINECART",
             "BUTTON",
-            "SIGN");
+            "SIGN",
+            "SHULKER_BOX");
     private List BLOCK_BLACKLIST = Arrays.asList(
             Material.ANVIL,
             Material.BEACON,
             Material.COMMAND_BLOCK,
             Material.CRAFTING_TABLE,
+            Material.CAULDRON,
             Material.BREWING_STAND,
             Material.DAYLIGHT_DETECTOR,
             Material.DISPENSER,
             Material.DROPPER,
             Material.ENCHANTING_TABLE,
             Material.ENDER_CHEST,
+            Material.FARMLAND,
             Material.FURNACE,
+//            Material.GRINDSTONE,
```

Comment:
You can uncommend all the blocks from 1.14/1.15, we're not really concerned with backwards compatibility since it's a plugin specifically designed for the server anyways